docs/index.md

```diff
@@ -29,6 +29,8 @@
   - Specification for new CLI - [WIP]
 
 - [task state, output, and trigger name rationalization](proposal-state-names.md)
+
+- [config file changes](proposal-config-changes.md)
```

Comment:
```suggestion
- [config file changes](proposal-config-changes.md) - A proposal for discussion at Cylc Workshop in Febuary 2020
```

Reply:
No need to add this - all things can be discussed anyway :grin: